webapp/channels/src/utils/post_utils.ts

```diff
@@ -723,6 +723,28 @@ export function getUserOrGroupFromMentionName(usersByUsername: Record<string, Us
     return '';
 }
 
+export function getUserOrGroupFromMentionName(
+    mentionName: string,
+    users: Record<string, UserProfile>,
+    groups: Record<string, Group>,
+    groupsDisabled?: boolean,
+    getMention = getMentionDetails,
+): [UserProfile | '', Group | ''] {
```

Comment:
I rather be `null` or `undefined` instead of an empty string

Reply:
Ok. I used an empty string since that's what the original logic was using. Didn't know if it was some convention or something. Would have preferred to use `null`, myself.

Reply:
ended up using `undefined` so didn't have to use explicit unions everywhere